packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -44,12 +44,13 @@ export default function ProfileMenu(): ReactElement {
         <Item>
           <Link href={user.permalink} passHref prefetch={false}>
             <a className="flex w-full">
-              <UserIcon className="mr-2 text-xl" /> Profile
+              <UserIcon filled={false} className="mr-2 text-xl" /> Profile
```

Comment:
Really sorry, I know you weren't the one who added the value for `text-xl` but can you check the equivalent value for it in terms of our guideline sizing? Thank you!

Reply:
Can you help me where I can check this?